playground/frontend/lib/components/dropdown_button/dropdown_button.dart

```diff
@@ -76,7 +90,7 @@ class _AppDropdownButtonState extends State<AppDropdownButton>
     return Container(
       height: kContainerHeight,
       decoration: BoxDecoration(
-        color: ThemeColors.of(context).greyColor,
+        color: widget.buttonColor ?? ThemeColors.of(context).greyColor,
```

Comment:
Maybe we should not introduce these overriding properties but always take colors from theme. Then if we need to override the theme colors, we wrap this widget in `Theme` widget to alter the theme. It would keep this widget simple.

Reply:
Thank you for the note, but I'd prefer to keep it as it is, it's much easier that way as for me.

Reply:
This seems important to me.

The reasons to only use Theme:
- Less tests to cover the widget. This is especially valuable because this line will be marked as covered even if only one of the options is used in tests. We do not plan to write widget tests for this, but we may at some point. And even without automated tests this means less manual tests.
- Less diverse calls of the constructor mean less chance of errors.
- No theme property is treated as special, this alone is easier to use.
- If this case repeats with overriding another property, that would mean more weight to 'only theme' solution, but by then we would already have an investment to 'theme or property' solution.